src/appleseed/renderer/meta/tests/test_lightsampler.cpp

```diff
@@ -39,14 +40,16 @@
 using namespace foundation;
 using namespace renderer;
 
-TEST_SUITE(Renderer_Kernel_Lighting_LightSampler)
+TEST_SUITE(Renderer_Kernel_Lighting_LightSamplers)
```

Comment:
Ideally we would have two test suites (and thus two source files), one per sampler type.